py3status/module.py

```diff
@@ -350,6 +350,35 @@ def set_module_options(self, module):
 
             self.i3s_module_options['separator_block_width'] = sep_block_width
 
+        for name in ('background', 'border'):
+            if name in mod_config:
+                param = mod_config[name]
+                if not isinstance(param, basestring):
+                    raise TypeError(
+                        'Invalid attribute `{}`, '
+                        'should be an int. Got `{}`'
+                        .format(name, param)
+                    )
+
+                self.py3_module_options['block'][name] = (
+                    Py3(self)._get_color(param)
+                )
+
+                if name != 'border':
+                    continue
```

Comment:
what's going on here?  I think you should just end the block here and unindent the following code seeing as you then screw the name variable over.

Reply:
`background` and `border` have identical code. I'm looping with two names. If users entered something... and it's background, `continue`. If it's border, then we make a quick list of borders, then start checking for inputs from users. Fine with screwing the name variable because I'm finished with `border` part (code is already added in block). Am working on getting border widths (int) at this time.

Reply:
Do you read what I write? 

I'm pretty sure it's wrong - yeah it runs but it is messy for no reason and is a maintenance nightmare.  If you are doing something clever where is the comment.

Have a fun evening I'm done :)

Reply:
You always say "What's going on here?" I explained it, but I know you know what's happening here... so why are you asking me that question? Maybe no question next time so I don't have to answer your question.

That's all I did... answering that question. I did read what you said. ;-)

Reply:
Done. :-) I keep tuples because it looks ugly / out of place if I list em all.

Reply:
ok so the code was ok except it had no comment.
github sometimes makes it hard to see the actual code

but this is much clearer so I prefer it.  Clarity is everything - make your code easy to read for idiot reviewers and your life is easier too